cmake/Platform/Windows/Packaging/Tests/test_installer.py

```diff
@@ -0,0 +1,248 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Pytest test configuration file.
+
+"""
+import pytest
+from pathlib import Path
+from subprocess import run, TimeoutExpired
+from time import sleep
+import json
+import os
+import shutil
+
+@pytest.fixture(scope="session")
+def installer_path(request):
+    return Path(request.config.getoption("--installer-path")).resolve()
+
+@pytest.fixture(scope="session")
+def install_root(request):
+    return Path(request.config.getoption("--install-root")).resolve()
+
+@pytest.fixture(scope="session")
+def project_path(request):
+    # MSBuild doesn't want us to output in the temp directory MSB8029
+    # so we use a pre-defined folder outside of %TMP%
+    #return tmpdir_factory.mktemp("TestProject")
+    path = Path(request.config.getoption("--project-path")).resolve()
+    yield path
+
+    # wait a few seconds for processes to quit
+    sleep(5)
+
+    shutil.rmtree(path)
+
+@pytest.fixture(scope="session")
+def cmake_path(install_root):
+    return install_root / 'cmake' / 'runtime' / 'cmake-3.20.2-windows-x86_64' / 'bin' / 'cmake.exe'
```

Comment:
The `'cmake-3.20.2-windows-x86_64'` part isn't supposed to exist in an installed build

Reply:
It was where the installer seemed to put the cmake.exe file in my tests, but you're right we'll definitely need to make this fixture capable of finding cmake.exe anywhere under `cmake/runtime`

Reply:
You're right about this - when I ran the nightly it did not have this subfolder and the cmake.exe was in `cmake\runtime\bin`  likely this is a problem with my install build creation process.

Reply:
I'm not sure why my local install task failed the move step, but I see it should have renamed this folder as part of the step here: https://github.com/o3de/o3de/blob/c2c8fb5b9a3d376eb355be41fb3b3e98e0101436/cmake/Platform/Windows/Packaging/PostInstallSetup.wxs#L53